package.json

```diff
@@ -25,6 +25,7 @@
   "readmeFilename": "README.md",
   "main": "index.js",
   "scripts": {
+    "pretest": "eslint *.js bin/*.js lib/*.js test/*.js",
```

Comment:
Is this better than `eslint --ignore-file .gitignore`? It might be, just asking.